docs/1-introduction/1.3.1-unix.md

```diff
@@ -40,6 +40,34 @@ mkdir liatrio_console_tutorial
 cd liatrio_console_tutorial
 ```
 
+## MKDIR and Directories
+
+Directories are folders in which you can organize your files. Infact, the directory you are in right now is a folder which holds all the other folders you need. If we navigate to our home directory with `cd ~` and then run `ls` we will be shown all the sub directories in our home (root) directory. We can create a directory on the command line! For those familiar with GUI based folder systems, this is like right clicking and pressing "Create New Folder", however its much simpler on the command line!. To create a directory, we can use the command `mkdir <directory name>`. Lets make a directory like this, `mkdir test_directory`. We can see that we now have a directory in our root `~/test_directory`. Now we an `cd test_directory` to enter into our newely created directory. But lets say we want to create a directory with sub directories within, all from our root (~) directory. Then we can add some cool extensions to this command to make it able to do this. To do this, we can run this command `mkdir -p parent/child/`. The -p flag (thats what these "additions" are called) lets the operating system know that we want the /child directory to be inside of the /parent directory. We can `cd parent` and `ls` and we can see that the /child directory shows up.
+
+Furthermore, another way to do this is by stringing together two `mkdir` commands with and `&&`. We can run `mkdir temp && temp/sub`, and the system will create the temp directory with the sub directory inside. This first runs the mkdir command on the left side of `&&`, which will make the directory 'temp'. Then it will run the command on the right side, which will make the directory 'sub' inside of the 'temp' directory that was just created.
```

Comment:
```suggestion
```

This paragraph seems way to directed at the specific quiz question. Though I do think covering logical operators like `&&` or `||` is some good information. I dont feel like this paragraph hits on the important component of those operators (ie: `&&` executes the right expression iff the left side returns a zero exit code. `||` executes the right expression only iff the left side exists with a non-zero exit code).

While I think this is useful `bash` knowledge this section in general is mission lots of other useful `bash` knowledge. Which is a can of worms I am not sure is worth going down. 

Personally I am also OK if our quizzes contain things that are new and encourage campers to look things up. 

TL'DR: lets remove this paragraph